docs/configuration/backends/ecs.md

```diff
@@ -129,6 +129,7 @@ Labels can be used on task containers to override default behaviour:
 | `traefik.protocol=https`                                  | override the default `http` protocol                                                     |
 | `traefik.weight=10`                                       | assign this weight to the container                                                      |
 | `traefik.enable=false`                                    | disable this container in Træfik                                                         |
+| `traefik.backend.port=80`                                 | override the default `port` value. Overrides `NetworkBindings` from Docker Container     |
```

Comment:
Could you please use [`LabelPort`](https://github.com/containous/traefik/blob/master/types/common_label.go#L10) instead of creating a new label.